prime-angular-frontend/src/app/modules/enrolment/pages/access-terms/access-terms.component.html

```diff
@@ -64,7 +78,7 @@
         <span>View Terms of Access</span>
       </button>
       <button mat-menu-item
-              [routerLink]="['enrolments', accessTerm.id]">
+              [routerLink]="[accessTerm.id, 'enrolment']">
```

Comment:
If you use the BaseEnrolmentPage you can use EnrolmentRoutes in the template so we don't have to find strings of the route in the templates if the routes change.